### PR TITLE
Editing of MTK doc 

### PR DESCRIPTION
**Pull request by ebgitelman**
## What Changed?

Copyediting of doc for consistency

### CODE REVIEW

**Review by drothery-edb — DISMISSED** [review]

**Review by drothery-edb — APPROVED** [review]

File: product_docs/docs/migration_toolkit/55/03_migration_methodology.mdx [hunk 1]
```diff
@@ -1,54 +1,52 @@
...
+5.  Migrate the data. For small data sets, use Migration Toolkit. If it's an Oracle migration into EDB Postgres Advanced Server, and the data set is large or if you notice slow data transfer, take advantage of one of the other data movement methods available:
 
     -   Use the EDB Postgres Advanced Server database link feature compatible with Oracle databases.
     -   If your data has BLOB or CLOB data, use the dblink_ora style database links instead of the Oracle style database links.
 
-    Both of these methods use the Oracle Call Interface (OCI) to connect to Oracle. After connecting, use an SQL statement to select the data from the 'linked' Oracle database and insert the data into the EDB Postgres Advanced Server database.
+    Both of these methods use the Oracle call interface (OCI) to connect to Oracle. After connecting, use an SQL statement to select the data from the 'linked' Oracle database and insert the data into the EDB Postgres Advanced Server database.
```
Comment (moiznalwalla) [inline product_docs/docs/migration_toolkit/55/03_migration_methodology.mdx:33]: Oracle capitalises this term https://www.oracle.com/in/database/technologies/appdev/oci.html.

File: product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx [hunk 2]
```diff
@@ -24,12 +24,12 @@ The following database product versions can be used with Migration Toolkit:
 -   MySQL 8.0
 -   Sybase Adaptive Server Enterprise 15.7
 
-Please contact your EnterpriseDB Account Manager or [sales@enterprisedb.com](mailto:sales@enterprisedb.com) if you require support for other database products.
+Contact your EnterpriseDB Account Manager or [sales@enterprisedb.com](mailto:sales@enterprisedb.com) if you require support for other database products.
 
 !!! Note
-    The Migration Toolkit has not been tested and does not officially support use with Oracle Real Application Clusters (RAC) and Exadata. However, Migration Toolkit may work when it is connected to a single persistent node. For more information, contact your EDB representative.
+    The Migration Toolkit wasn't tested with and doesn't officially support use with Oracle Real Application Clusters (RAC) and Exadata. However, Migration Toolkit might work when it is connected to a single persistent node. For more information, contact your EDB representative.
```
Comment (moiznalwalla) [inline product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx:30]: Should we replace this with "isn't"?

"Wasn't" might imply that it has now been tested.

File: product_docs/docs/migration_toolkit/55/03_migration_methodology.mdx (hunk 1 above)
Comment (moiznalwalla) [reply product_docs/docs/migration_toolkit/55/03_migration_methodology.mdx:33]: I usually face a similar dilemma on whether to capitalise Azure resources or not. Even Azure docs seems to capitalise them inconsistently in some of their docs. 

For example: in (https://docs.microsoft.com/en-us/azure/private-link/create-private-endpoint-portal) the term private endpoint is inconsistently capitalised.

File: product_docs/docs/migration_toolkit/55/07_invoking_mtk.mdx [hunk 3]
```diff
@@ -98,7 +92,7 @@ Where:
 
 `-targetdbtype target_type`
 
-`target_type` specifies the server type of the target database. `target_type` is case-insensitive. By default, `target_type` is `enterprisedb`. `target_type` may be one of the following values:
+`target_type` specifies the server type of the target database. `target_type` is not case sensitive. By default, `target_type` is `enterprisedb`. `target_type` can be one of the following values:
```
Comment (moiznalwalla) [inline product_docs/docs/migration_toolkit/55/07_invoking_mtk.mdx:95]: Should we use angular brackets here?

File: product_docs/docs/migration_toolkit/55/03_migration_methodology.mdx (hunk 1 above)
Comment (ebgitelman) [reply product_docs/docs/migration_toolkit/55/03_migration_methodology.mdx:33]: Okay, thanks. Will fix.

File: product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx (hunk 2 above)
Comment (ebgitelman) [reply product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx:30]: good edit!

File: product_docs/docs/migration_toolkit/55/07_invoking_mtk.mdx (hunk 3 above)
Comment (ebgitelman) [reply product_docs/docs/migration_toolkit/55/07_invoking_mtk.mdx:95]: Yes, and if you look at the command, it says <dbtype> where it should say <source_type>. I will fix that and add the angle brackets to the explanatory text for that as well.

**Pull request merged**